first-step/src/main/java/org/elearning/portfolio/services/UserService.java

```diff
@@ -0,0 +1,46 @@
+package org.elearning.portfolio.services;
+
+
+import java.util.List;
+
+import org.elearning.portfolio.user.*;
+import org.elearning.portfolio.message.*;
+
+
+public class UserService{
+
+    private  UserDAOImpl user  ;
+
+    public UserService(UserDAOImpl user){
+        this.user = user;
+    }
+
+    public void setUser(UserDAOImpl user){this.user = user;}
+
+    public List <User>  getUser(Integer userId){return user.getUser(userId) ;}
```

Comment:
A to ciekawe getUser - liczba pojedyncza a zwraca listę - ta metoda jest niejasna. Skoro getUser to powinno zwrócić jeden rekord, a jeżeli wystąpi więcej, no to raczej jest błąd.